src/CLICommand/common_context.ts

```diff
@@ -78,4 +81,19 @@ export class CommonContext {
 		const value = this.options[parameterName];
 		return value;
 	}
+
+	/**
+	 * @param {ParameterName} parameterName
+	 * @returns {string | undefined}
+	 * Returns the string value for the specific parameter; returns undefined if not set
+	 */
+	public getRequiredParameterValue(parameterName: ParameterName): string | undefined {
+		const value = this.options[parameterName];
+		if (!value) {
+			throw new Error(`Parameter ${parameterName} wasn't provided!`);
```

Comment:
Might want to say `Required parameter...`

Reply:
Updates the logged message: 8353bff0976349bd9813d46e92b3de6fbfce3684